packages/react/src/components/ColorPicker/ColorPicker.test.tsx

```diff
@@ -210,62 +209,67 @@ describe('ColorPicker', () => {
       hueAriaLabel: 'custom hue',
     };
 
-    wrapper = mount(<ColorPicker color="#FFFFFF" strings={customStrings} />);
+    const { getAllByRole } = render(<ColorPicker color="#FFFFFF" strings={customStrings} />);
 
-    const tableHeaders = wrapper.find('thead td');
+    const tableHeaders = Array.from(getAllByRole('group')[1].firstElementChild!.firstElementChild!.children);
     tableHeaders.forEach((node, index) => {
-      expect(node.text()).toEqual(fields[index]);
+      expect(node.textContent).toEqual(fields[index]);
     });
 
     // Check for the aria strings in the HTML of the corresponding components (simplest way
     // to verify ColorPicker is passing custom values through correctly)
-    const rectangleHtml = wrapper.find(ColorRectangleBase).html();
-    expect(rectangleHtml).toContain(customStrings.svAriaLabel);
-    expect(rectangleHtml).toContain(customStrings.svAriaDescription);
-    expect(rectangleHtml).toContain(customStrings.svAriaValueFormat);
+    const rectangleHtml = getAllByRole('slider')[0];
+    expect(rectangleHtml.getAttribute('aria-label')).toEqual(customStrings.svAriaLabel);
+    expect(rectangleHtml.firstElementChild!.textContent).toEqual(customStrings.svAriaDescription);
+    expect(rectangleHtml.getAttribute('aria-valuetext')).toEqual(customStrings.svAriaValueFormat);
 
-    const sliders = wrapper.find(ColorSliderBase);
-    expect(sliders.at(0).html()).toContain(customStrings.hueAriaLabel);
-    expect(sliders.at(1).html()).toContain('Custom Alpha');
+    const sliders = getAllByRole('slider');
+    expect(sliders[1].getAttribute('aria-label')).toEqual(customStrings.hueAriaLabel);
+    expect(sliders[2].getAttribute('aria-label')).toEqual('Custom Alpha');
   });
 
   it('uses default aria label', () => {
-    wrapper = mount(<ColorPicker color="#abcdef" />);
-    expect(wrapper.getDOMNode().getAttribute('aria-label')).toBe(
+    const { getAllByRole, rerender } = render(<ColorPicker color="#abcdef" />);
+    expect(getAllByRole('group')[0].getAttribute('aria-label')).toBe(
```

Comment:
Is `group` just the wrapper element? If so I think something like `container.firstElementChild` might be better.

Reply:
Just made the change for this but just wondering, what makes `container.firstElementChild` better than `getAllByRole('group')[0]` in this case?

Reply:
Oh sorry, I was thinking "group" was some implied role, but now I see it's explicitly on the root element of the ColorPicker--so that's probably fine to get it by that role if you add a comment about what it is. (Or just leave it if you don't want to bother reverting changes. Sorry about the confusion, I should have looked at the component code!)

Reply:
Ah no worries! Just reverted back to getting by role and declaring it in a `colorPickerRoot` variable for clarity.